Makefile

```diff
@@ -33,22 +33,6 @@ test:
 
 int: constd
 	go test -v ./tests --tags=integration
-# CONSTD TARGETS
-
-constd:
-	go build -o .bin/constd -tags wasmtime,netgo ./constd
```

Comment:
What do we want to do with the `int`, `constd/metal/otel/hc`, and `constd/metal/otel/collector` targets?

Reply:
That's a @javorszky question, probably move it over to the deltav repo once that's updated.

Reply:
Yeah, those are all integration tests. We shall move them over to the deltav repo, though I haven't found that. It's on my list. For now I've created an issue in sat and assigned to myself: https://github.com/suborbital/sat/issues/146